docs/forms_and_cases.rst

```diff
@@ -127,17 +127,6 @@ For more details see:
 * :code:`corehq.form_processor.interfaces.dbaccessors.LedgerAccessors`
 
 
-Branching
```

Comment:
If there are still other usages of this function then we should keep the docs around.

Reply:
My thought was that this function is now deprecated and no new code should use it. This PR is the beginning of a multi-step process to remove all references to it. I guess I could wait to remove the docs until removing the last reference. Do you think that would be better?

Reply:
No, you're right - we shouldn't need this any more.